docs/integrations/sources/harvest.md

```diff
@@ -1,12 +1,54 @@
 # Harvest
 
-## Overview
+This page contains the setup guide and reference information for the Harvest source connector.
 
-The Harvest connector can be used to sync your Harvest data. It supports full refresh sync for all streams and incremental sync for all streams except of Expense Reports streams which are: Clients Report, Projects Report, Categories Report, Team Report. Incremental sync is also now available for Company stream, but it always has only one record.
+## Prerequisites
+
+See [docs](https://help.getharvest.com/api-v2/authentication-api/authentication/authentication/) for more details.
+
+## Setup guide
+### Step 1: Set up Harvest
+
+This connector supports only authentication with API Key. To obtain API key follow the instructions below:
+
+1. Go to Account Settings page;
+2. Under Integrations section press Authorized OAuth2 API Clients button;
+3. New page will be opened on which you need to click on Create New Personal Access Token button and follow instructions.
+
+## Step 2: Set up the Harvest connector in Airbyte
+
+### For Airbyte Cloud:
+
+1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+2. In the left navigation bar, click **Sources**. In the top-right corner, click **+new source**.
+3. On the Set up the source page, enter the name for the Harvest connector and select **Harvest** from the Source type dropdown.
+4. For Airbyte Cloud, click **Authenticate your Harvest account** to sign in with Harvest and authorize your account.
+5. Enter your `account_id` 
+6. Enter the `replication_start_date` you want your sync to start from
+7. Click **Set up source**
+
+### For Airbyte OSS:
+1. Navigate to the Airbyte Open Source dashboard
+2. Set the name for your source 
+3. Enter your `api_token`
+4. Enter your `account_id` 
+5. Enter the `replication_start_date` you want your sync to start from
+6. Click **Set up source**
+
+## Supported sync modes
+
+The Harvest source connector supports the following[ sync modes](https://docs.airbyte.com/cloud/core-concepts#connection-sync-modes):
```

Comment:
Can we make this piece `following[ sync modes]` to be ` following [sync modes]() `?)

Reply:
updated